client/my-sites/stats/summary/index.jsx

```diff
@@ -369,7 +358,28 @@ class StatsSummary extends Component {
 				<NavigationHeader className="stats-summary-view" navigationItems={ navigationItems } />
 
 				<div id="my-stats-content" className="stats-summary-view stats-summary__positioned">
-					{ summaryViews }
+					<StatsGlobalValuesContext.Consumer>
+						{ ( isInternal ) => (
+							<>
+								{ this.props.context.params.module === 'utm' &&
+									( supportsUTMStats || isInternal ) && (
+										<>
+											{ this.renderSummaryHeader( path, statType, false, moduleQuery ) }
+											<StatsModuleUTM
+												siteId={ siteId }
+												period={ this.props.period }
+												query={ moduleQuery }
+												summary
+											/>
+										</>
+									) }
+								{ this.props.context.params.module === 'utm' &&
+									! supportsUTMStats &&
+									! isInternal && <div>{ translate( 'This path is not available.' ) }</div> }
+							</>
+						) }
+					</StatsGlobalValuesContext.Consumer>
+					{ ! this.props.context.params.module !== 'utm' && summaryViews }
```

Comment:
I'm having trouble parsing this logic. Are the two `!` operators intentional here?

Reply:
Dognose's proposal simplified the conditions.